addons/indicators/stringtable.xml

```diff
@@ -65,6 +65,12 @@
                 <Polish>Ustawia rozmiar znaczników.</Polish>
                 <Portuguese>Define o tamanho dos indicadores</Portuguese>
             </Key>
+            <Key ID="STR_dui_indicators_useACENametagsRange">
+                <English>Use ACE Nametags Range</English>
+            </Key>
+            <Key ID="STR_dui_indicators_useACENametagsRange_desc">
+                <English>Matches indicator transparency to ACE Nametags transparency</English>
```

Comment:
To avoid confusion, maybe add another sentence that this will overwrite the range setting